src/RestEaseUnitTests/ImplementationBuilderTests/RawQueryParameterTests.cs

```diff
@@ -59,16 +65,43 @@ public void AddsRawQueryParam()
 
             implementation.FooAsync("&test=test2");
 
-            Assert.NotNull(requestInfo.RawQueryParameter);
-            Assert.Equal("&test=test2", requestInfo.RawQueryParameter.SerializeToString(null));
+            Assert.NotNull(requestInfo.RawQueryParameters);
+            Assert.Equal("&test=test2", requestInfo.RawQueryParameters.SerializeToString(null));
```

Comment:
I think we should be testing that `RawQueryParameters` contains a single element, and then test the value of that element